operations/patch.go

```diff
@@ -83,6 +74,15 @@ func Patch() cli.Command {
 				return errors.Wrap(err, "problem loading configuration")
 			}
 
+			uncommittedChanges, err := gitUncommittedChanges()
+			if err != nil {
+				return errors.Wrap(err, "can't test for uncommitted changes")
+			}
+
+			if (!params.Uncommitted && !conf.UncommittedChanges) && uncommittedChanges {
+				grip.Infof("Uncommitted changes are omitted from patches by default.\nUse the '--%s, -u' flag or the 'patch_uncommitted_changes' field in the Evergreen yaml file to include uncommitted changes.", uncommittedChangesFlag)
```

Comment:
"or set 'patch_uncommitted_changes: true' in your ~/.evergreen.yml file"

Reply:
done.